evap/staff/tests/test_views.py

```diff
@@ -912,66 +913,42 @@ def test_remove_responsibility(self):
 
         self.assertIn("No responsible contributors found", page)
 
-    def test_approve_without_saving(self):
+    def test_last_modified_user(self):
         """
-            Tests whether the button "Approve" does actually only approve the
-            course without saving changes to the course.
+            Tests whether the button "Save and approve" does only change the
+            last_modified_user if changes were made.
         """
-        old_name_de = self.course.name_de
-        old_vote_start_datetime = self.course.vote_start_datetime
-        old_vote_end_date = self.course.vote_end_date
-
-        new_name_de = "Test"
-        new_vote_start_datetime = "2018-01-01 00:00"
-        new_vote_end_date = "2099-12-31"
+        test_user = mommy.make(UserProfile, username='approve_test_user', groups=[Group.objects.get(name='Staff')])
 
-        self.assertNotEqual(old_name_de, new_name_de)
-        self.assertNotEqual(old_vote_start_datetime, new_vote_start_datetime)
-        self.assertNotEqual(old_vote_end_date, new_vote_end_date)
+        old_last_modified_user = self.course.last_modified_user
+        old_state = self.course.state
+        self.assertEqual(old_last_modified_user.username, self.user.username)
+        self.assertEqual(old_state, "new")
 
-        page = self.get_assert_200('/staff/semester/{}/course/{}/edit'.format(self.course.semester.pk, self.course.pk), user="staff")
+        page = self.get_assert_200('/staff/semester/{}/course/{}/edit'.format(self.course.semester.pk, self.course.pk), user=test_user.username)
         form = page.forms["course-form"]
-
-        form["vote_start_datetime"] = new_vote_start_datetime
-        form["vote_end_date"] = new_vote_end_date
-        form["name_de"] = new_name_de
+        # approve without changes
         form.submit(name="operation", value="approve")
 
         self.course = Course.objects.get(pk=self.course.pk)
-        self.assertEqual(self.course.name_de, old_name_de)
-        self.assertEqual(self.course.vote_start_datetime, old_vote_start_datetime)
-        self.assertEqual(self.course.vote_end_date, old_vote_end_date)
+        self.assertEqual(self.course.last_modified_user, old_last_modified_user)
         self.assertEqual(self.course.state, 'approved')
```

Comment:
can you assert the other values didn't change?

Reply:
is it okay to only test some values (e.g. name_de, vote_start_datetime, vote_end_date)?